apps/fabric-website/homepage.htm

```diff
@@ -1,6 +1,6 @@
 <div
   id="metadata"
-  title="Home - Office UI Fabric"
+  title="Home - Fluent UI"
```

Comment:
This will need to be updated in the internal version as well (this file is just a mirror).

Also, the next line "The official front-end framework for building experiences that fit seamlessly into Office and Office 365." needs to be updated (and check for anywhere else something like that occurs).

Reply:
ADO task added for looking for/fixing "office" references